preview-docs/OperationsCatalog.md

```diff
@@ -0,0 +1,25 @@
+
+# Proposed use cases
+
+1. **Save and organize operations as collections:** A collection is a set of operations. Developers will be able to save the operations that they write in Studio for later use and easy access and organize them as collections. They can create, update, delete and read the collections of operations. Collections will have documentation that can be used to explain their purpose. Comments on collections for collaboration will also be made available. 
+2. **Share with other members of the team:** Developers can share the collection/s of operations with other members of the team that have at least read access to the graph. For public graphs, sharing can be possible outside of the members of the Studio Org. Sharing of collections is made possible by:
+    1. Share the collection directly with other members of the org that have access to Studio: The shared collection will be visible in the operations catalog of each member that it is shared with. Any user with the permission to edit the collection can make changes to the collection or any of the operation. Sharing can also be done as view-only, where the user with view-only permission cannot make any changes to the collection.
+    2. Share the collection with a link: A unique link for that collection will be generated that can be distributed for sharing. Anyone who has access to the link will be able to view the collections. Link sharing can be used for sharing a collection outside of the Studio org. Link sharing only gives view-only access to the collections. The link is always up to date with the changes made to the collection.
+    3. Share the collection with a run in Explorer button: Using an explorer button, operations collections can be embedded for use in documentation. Using this button, a developer can either import the collection and use it in their own IDE or they can open the collection in Apollo Sandbox to execute. The run button for collection is always up to date with the changes made to the collection.
+3. **Reference and download:** A GraphQL document that represents the collection can also be easily exported for distribution. A particular operation or collection of operations can also be used in markdown for inline rendering. 
+4. **Production data access:** Certain operations or collections can be marked as “secure”, meaning that they are good for pulling production data, allowing non-technical users who want to query graph data easy access to parameterize those operations. A specific list of users or a user with a secret token can query data using secure operations/collections.
```

Comment:
Would a collection potentially span graphsn and variants? So you could save a collection that included some operations to prod, some operations to staging, and some operations to another graph entirely?